src/node_api.cc

```diff
@@ -113,22 +113,22 @@ static v8::Local<v8::Function> V8LocalFunctionFromJsValue(napi_value v) {
 
 // Wrapper around v8::Persistent that implements reference counting.
 class Reference {
- public:
+ protected:
```

Comment:
Why not `private`? 'protected' implies there might be a subclass, but there isn't.